### PR TITLE
[stable/cluster-autoscaler] - imagePullSecrets support #9584

### LINKED ISSUE #9584: [stable/cluster-autoscaler] - imagePullSecrets support
Is this a BUG REPORT or FEATURE REQUEST? (choose one):
Feature request

Version of Helm and Kubernetes:
Helm 2.11.0
Kubernetes 1.10.3-eks

Which chart:
cluster-autoscaler version 0.9.0

What happened:
Currently the helm chart doesn't allow for specifying imagepullSecrets so I had to modify the chart locally to use it.

What you expected to happen:
I would like to be able to provide image.pullSecrets as part of values.yaml

How to reproduce it (as minimally and precisely as possible):
Try using cluster-autoscaler image from a docker registry that requires auth

### PR DESCRIPTION
**Pull request by nxf5025**
Signed-off-by: Nick Fisher <nxf5025@gmail.com>

#### What this PR does / why we need it:
Added configuration option for imagePullSecrets in cluster-autoscaler chart.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #9584 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/cluster-autoscaler/README.md [hunk 1]
```diff
@@ -121,6 +121,7 @@ Parameter | Description | Default
 `image.repository` | Image | `k8s.gcr.io/cluster-autoscaler`
 `image.tag` | Image tag  | `v1.2.0`
 `image.pullPolicy` | Image pull policy  | `IfNotPresent`
+`image.pullSecrets` | Image pull secrets  | `None`
```
Comment (Dean-Coakley) [inline stable/cluster-autoscaler/README.md]:
```suggestion
`image.pullSecrets` | Image pull secrets  | `[]`
```

Comment (nxf5025) [reply stable/cluster-autoscaler/README.md]: PR updated

File: stable/cluster-autoscaler/values.yaml [hunk 2]
```diff
@@ -49,6 +49,7 @@ image:
   repository: k8s.gcr.io/cluster-autoscaler
   tag: v1.12.0
   pullPolicy: IfNotPresent
+  pullSecrets: []
```
Comment (lachie83) [inline stable/cluster-autoscaler/values.yaml:52]: Can you add some comments to reference on how this to be used eg. https://github.com/helm/charts/blob/master/stable/searchlight/values.yaml#L17-L23

**Comment by linki** [comment]
/unassign @linki

(I'm not a maintainer anymore)

**Pull request closed without merging**